### PR TITLE
SNOW-620437 changed Core from 2.0 to 6.0

### PR DESCRIPTION
**Pull request by sfc-gh-mheavin**

### CODE REVIEW

**Review by sfc-gh-mpeters — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The Snowflake .NET connector supports the the following .NET framework and libra
 
 - .NET Framework 4.7.2
 - .NET Framework 4.7.3
-- .NET Core 2.0
+- .NET Core 6.0
```
Comment (sfc-gh-dyoshinaga) [inline README.md:12]: This file also contains some examples that appear to refer to .NET Core 2.0:

```
dotnet test -f netcoreapp2.0
```

```
OpenCover.4.6.519\tools\OpenCover.Console.exe ... output:"netcoreapp2.0_coverage.xml" ...
```

Do these references need to be updated as well?

**Review by sfc-gh-dyoshinaga — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sfc-gh-mheavin) [reply README.md:12]: Fixed. Thanks.

File: README.md [hunk 2]
```diff
@@ -564,5 +564,4 @@ This CVE has been reported in systems.text.regularexpressions.dll which is used
 	Snowflake has identified an issue where the driver is globally enforcing TLS 1.2 and certificate revocation checks with the .NET Driver v1.2.1 and earlier versions.  
 	Starting with v2.0.0, the driver will set these locally.  
 	
-	Note that the driver is now targeting .Net framework 4.7.2.  
-	When upgrading to v2.0.0, you might also need to run "Update-Package -reinstall" to update the dependencies.
+  Note that the driver is now targeting .NET 6.0. When upgrading to {release version that targets .NET 6.0}, you might also need to run “Update-Package -reinstall” to update the dependencies.
```
Comment (sfc-gh-dyoshinaga) [inline README.md:567]: > When upgrading to {release version that targets .NET 6.0},

Do the curly braces represent some sort of variable?

If not, could this just be written as "When upgrading to a release version ..." without the curly braces?

**Review by sfc-gh-dyoshinaga — APPROVED** [review]
Thank you for making these fixes!

**Pull request merged**